catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/adapter/BrickAdapter.java

```diff
@@ -273,6 +273,15 @@ public boolean removeItems(List<BrickBaseType> items) {
 		return false;
 	}
 
+	public List<Integer> getPositionsOfItems(List<Brick> bricks) {
```

Comment:
I'm not entirely sure if this is the best way to do it. I'll have to look into how you implemented the whole drawing methods in more detail. 

I'm not saying this is wrong, I just want to check if there are any performance issues with the fragment overdraw. I'll get back to you about it.

Reply:
seems ok.